nestjs/src/config/config.service.ts

```diff
@@ -48,6 +49,7 @@ export class ConfigService {
         clientSecret: conf.get('RSSHCOOL_AUTH_GITHUB_CLIENT_SECRET'),
         callbackUrl: conf.get('RSSHCOOL_AUTH_GITHUB_CALLBACK'),
         scope: ['user:email'],
+        activityWebhookSecret: conf.get('process.env.ACTIVITY_WEBHOOK_SECRET', 'activity-webhook'),
```

Comment:
@stpnvntn  please change it to `RSSHCOOL_WEBHOOK_ACTIVITY_SECRET` and add it to `docker-compose.yml` file. I will make necessary changes on infra side to support it

Reply:
@apalchys maybe `RSSHCOOL_AUTH_GITHUB_WEBHOOK_ACTIVITY_SECRET `? 
To keep it consistent with all GitHub vars?

Reply:
pushed with `RSSHCOOL_AUTH_GITHUB_WEBHOOK_ACTIVITY_SECRET `